osquery/tables/applications/browser_chrome.cpp

```diff
@@ -19,18 +25,38 @@ namespace tables {
 #endif
 
 QueryData genChromeExtensions(QueryContext& context) {
-  fs::path chromePath;
 
   /// Each home directory will include custom extensions.
   if (isPlatform(PlatformType::TYPE_WINDOWS)) {
-    chromePath = "\\AppData\\Local\\Google\\Chrome\\User Data\\%\\Extensions\\";
+    chrome_path =
+        "\\AppData\\Local\\Google\\Chrome\\User Data\\%\\Extensions\\";
+  } else if (isPlatform(PlatformType::TYPE_OSX)) {
+    chrome_path = "/Library/Application Support/Google/Chrome/%/Extensions/";
+  } else {
+    chrome_path = "/.config/google-chrome/%/Extensions/";
+  }
+
+  if (isPlatform(PlatformType::TYPE_WINDOWS)) {
+    brave_path = "\\AppData\\Roaming\\brave\\Extensions\\";
   } else if (isPlatform(PlatformType::TYPE_OSX)) {
-    chromePath = "/Library/Application Support/Google/Chrome/%/Extensions/";
+    brave_path =
+        "/Library/Application "
+        "Support/BraveSoftware/Brave-Browser/%/Extensions/";
   } else {
-    chromePath = "/.config/google-chrome/%/Extensions/";
+    brave_path = "/.config/BraveSoftware/Brave-Browser/%/Extensions/";
   }
 
-  return genChromeBasedExtensions(context, chromePath);
+  if (isPlatform(PlatformType::TYPE_WINDOWS)) {
+    chromium_path = "\\AppData\\Local\\Chromium\\Extensions\\";
+  } else if (isPlatform(PlatformType::TYPE_OSX)) {
+    chromium_path = "/Library/Application Support/Chromium/%/Extensions/";
+  } else {
+    chromium_path = "/.config/chromium/%/Extensions/";
+  }
+
+  std::vector<fs::path> chrome_paths{chrome_path, brave_path, chromium_path};
+
+  return genChromeBasedExtensions(context, chrome_paths);
```

Comment:
You can instead write this as:
```
  return genChromeBasedExtensions(context,
                                  {chrome_path, brave_path, chromium_path});
```

And not use the temporary `chrome_paths` name.